x-pack/legacy/plugins/upgrade_assistant/public/np_ready/application/components/tabs/checkup/deprecations/_index.scss

```diff
@@ -1,5 +1,5 @@
-@import './cell';
-@import './reindex/index';
+@import 'cell';
+@import 'reindex/index';
 
 .upgDeprecations {
```

Comment:
There shouldn't be any actual style declarations in an `_index.scss` file. Can you move these to a component-specific file?